File: starter/Day4/src/DrawTrees.java
import acm.graphics.*;
import acm.program.GraphicsProgram;
import java.awt.Color;

public class DrawTrees extends GraphicsProgram {

	private static final int TRUNK_WIDTH = 20;
	private static final int TRUNK_HEIGHT = 20;
	private static final int LEAVES_RADIUS = 30;
	
	public void run() {
		drawTree(50, 150);
		drawTree(100, 300);
		drawTree(500, 200);
		drawTree(300, 250);
		drawTree(700, 310);
	}

	/** 
	 * Method: Draw Tree
	 * -----------------
	 * The draw tree method draws a tree with the base of 
	 * its trunk centered at the provided (x, y) position:
	 */
	private void drawTree(int x, int y) {
		// your code here...
	}
}

File: templates/en/projects/madmethods/starter/draw_trees.py
"""
File: draw_trees.py
----------------
This program draws several trees on a canvas.
"""

from graphics import Canvas

# These constants tell the graphics canvas how big to be.  You can ignore them.
CANVAS_WIDTH = 800
CANVAS_HEIGHT = 400

TRUNK_WIDTH = 20  # The width of each tree trunk. Use this!
TRUNK_HEIGHT = 80  # The height of each tree trunk. Use this!
LEAVES_RADIUS = 30  # The radius of the tree "leaf". Use this!


def main():
    canvas = Canvas(CANVAS_WIDTH, CANVAS_HEIGHT)
    canvas.set_canvas_title("Draw Trees")

    draw_tree(canvas, 50, 150)
    draw_tree(canvas, 100, 300)
    draw_tree(canvas, 500, 200)
    draw_tree(canvas, 300, 250)
    draw_tree(canvas, 700, 310)

    canvas.mainloop()


def draw_tree(x, y):
    """
    The draw_tree method draws a tree with the base of its trunk
    centered at the provided (x, y) position.
    """
    # TODO: Your code here


if __name__ == "__main__":
    main()
